README.md

```diff
@@ -22,7 +63,861 @@ Or install it yourself as:
 
```

Comment:
I think it would be better to add a Glossary section with the terminolody you are intending to use before the Usage section.

Reply:
e.g. Command, Contract, Precondition, Monad, Aggregate, ...

Reply:
Half of it is exactly explained in the doc, and another one is a common developers knowledge that can be googled.

Reply:
I understand that the knowledge itself is found across the documentation. I was saying that it think it could be better to centralize it in one section before the usage so that there is no going back and forth in the documentation trying to re-read what was written, that's all :D